examples/netlogo/README.md

```diff
@@ -135,3 +135,8 @@ The content of the code tab is created by simply drawing together all blocks sho
 ; ==> Go.
 ; ==> Submodels.
 ```
+
+### NetLogo file
+
+In the main `nlogo` file, we only "include" an `nls` file to allow for the reverse mode.
```

Comment:
Maybe explicitly mention that this also enables using NetLogo's RAD tools to modify the project and its UI?

Reply:
Added this at the very end:

This separation of model code and user interface also allows to edit the model's UI elements in NetLogo's GUI builder tool, while using Literate Programming for the code.